CONTRIBUTING.md

```diff
@@ -122,6 +122,11 @@ We recommend you install GHC and Cabal using [GHCup].
 
 If you'd like to use a different version of GHC, you can find the list of versions that we test with in [build-vehicle.yml]. However, be aware that building the Python bindings _requires_ [our preferred version](#the-preferred-version-of-ghc).
 
+Some of the Haskell libraries used by Vehicle also required `gmp` arithmetic library to be installed on the system. If it is not installed on your system you can install it, e.g. on Linux by
+   ```sh
+   sudo apt-get install libgmp3-dev
```

Comment:
When are folks encountering this issue? It shouldn't be with a Python install.

Reply:
This install command is specific to Ubuntu, which means it's not all that useful. Furthermore, most Haskell installation instructions inside instructions for installing gmp, or do it automatically.

Reply:
> When are folks encountering this issue? It shouldn't be with a Python install.

This was installing it from source on my new work laptop...